freeipa/src/main/resources/schema/app/20211214100402_CB-15466_cleanup_operation_indexes.sql

```diff
@@ -0,0 +1,19 @@
+-- // CB-15466 cleanup operation indexes
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE operation DROP CONSTRAINT IF EXISTS syncoperation_operationid_key;
+
+DROP INDEX IF EXISTS syncoperation_operationid_key;
+
+DROP INDEX IF EXISTS syncoperation_id_idx;
+
+DROP INDEX IF EXISTS syncoperation_operationid_idx;
+
+DROP INDEX IF EXISTS syncoperation_accountid_endtime_idx;
+
+CREATE INDEX IF NOT EXISTS operation_start_end_type_idx ON operation (starttime, endtime, operationtype)
+
+-- //@UNDO
+-- SQL to undo the change goes here.
```

Comment:
Does not need the undo part?

Reply:
I'm adding the undo part for the new index, but not for cleaning the old ones, as they are not useful in any way or indexing the same column(s)